test-apps/display-test-app/README.md

```diff
@@ -160,6 +163,8 @@ You can use these environment variables to alter the default behavior of various
   * See TileAdmin.Props.minimumSpatialTolerance.
 * SVT_NO_EXTERNAL_TEXTURES
   * If defined, the backend will embed all texture image data directly in the tiles.
+* SVT_LOG_FRAMESTATS

```

Comment:
This seems like it'd be more useful as a toggle key-in.

Reply:
I removed the environment variable and added a dta keyin.  `dta framestats`